Update README (#339)
vmware-tanzu/tanzu-toolkit-for-visual-studio: README.md

- [Remotely debug](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/wiki/Remote-Debugging-TAS-Apps) apps running on TAS within Visual Studio

## Install

> To install this extension, you will need Visual Studio version 2022 or 2019. This extension can be downloaded within the IDE by searching "Tanzu" in the "Manage Extensions" window

Reviewer: ```suggestion
To install this extension, you will need Visual Studio version 2022 or 2019. This extension can be downloaded within the IDE by searching "Tanzu" in the "Manage Extensions" window.
```

![Extension manager showing Tanzu Toolkit for Visual Studio 2022](https://user-images.githubusercontent.com/22666145/169391289-f8ebe6ef-8209-47ae-86e2-5ef293b19cf2.png)

> This extension can also be installed outside of the IDE by first downloading the appropriate `.vsix` file, then double-clicking it to prompt an installation into your instance of Visual Studio.

Reviewer: ```suggestion
This extension can also be installed outside of Visual Studio by first downloading the appropriate `.vsix` file, then double-clicking it to prompt an installation.
```

        - through the website: [VS 2022](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2022), [VS 2019](https://marketplace.visualstudio.com/items?itemName=TanzuNETExperience.TanzuToolkitForVisualStudio2019)

>       - vsixgallery.com: [VS 2022](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2022.ff7b6f3e-0410-4ff9-a40a-a719ee9da901), [VS 2019](https://www.vsixgallery.com/extension/TanzuToolkitForVisualStudio2019.ff7b6f3e-0410-4ff9-a40a-a719ee9da901)

Reviewer 2: We should separate this out to a separate section or indicate that these are the "latest" builds and probably won't match the VS Marketplace or releases section

> Alternatively, latest builds can be found on vsixgallery.com, though these may contain features still under development:

Reviewer: ```suggestion
Alternatively, latest builds can be found on [vsixgallery.com](https://www.vsixgallery.com/), though these may contain features still under development:
```

> Once you've downloaded `Tanzu.Toolkit.VisualStudio.vsix`, double-click it to prompt an installation into your instance of Visual Studio.

Reviewer 2: This won't apply if you install through the IDE

Reviewer 2: ```suggestion
If the process you selected involves saving `Tanzu.Toolkit.VisualStudio.vsix` to disk, double-click it to prompt an installation into your instance of Visual Studio.
```

Author: Oops didn't see this suggestion

Author: I rephrased the whole section in this commit: [6f26e8e](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/339/commits/6f26e8ec1e9c6626811da2f18c791db4deb3c9a7) -- better?

## Uninstall

> To uninstall this extension from Visual Studio, visit the "Extension Manager" found under the `Extensions` menu.

Reviewer: ```suggestion
To uninstall this extension from Visual Studio, visit the "Extension Manager" found under the `Extensions` menu.
This extension can be uninstalled from Visual Studio using  "Manage Extensions" window.
```

Author: Changed in cd43144
